docs/docs/css-in-js.md

```diff
@@ -2,7 +2,7 @@
 title: Enhancing Styles with CSS-in-JS
 ---
 
-CSS-in-JS refers to an approach where styles are written in JavaScript instead of in external CSS files to easily scope styles in components, eliminate dead code, encourage faster performance and dynamic styling, and more.
+CSS-in-JS refers to an approach where styles are written in JavaScript instead of in external CSS files to scope styles in components, eliminate dead code, encourage faster performance and dynamic styling, and more.
```

Comment:
I would rephrase this, "to make it easier to scope styles in components", for example.